jupyter-book/evaluation/evaluation_quiz_m7_04.md

```diff
@@ -11,10 +11,16 @@ What is the default score in scikit-learn when using a classifier?
 +++
 
 ```{admonition} Question
-Which classifier's method should you call to get the predictions required
-by the function `precision_recall_curve`?
+Other than the decision threshold, metrics such as recall and precision also
+depend on the regularization parameters. Assuming that class "1" (in red) is
+the positive class, use the following figures to select the which statements
+are true in this particular logistic regression model
 
-- a) `predict(X)`
-- b) `decision_function(X)`
-- c) `predict_proba(X)`
+![Precision-recall C=3e-3](../../figures/evaluation_quiz_precision_recall_C0.003.svg)
```

Comment:
From the discussion that we had with @ogrisel, it could be good:

- put link to the documentation in scikit-learn for precision and recall
- add the relation between C and high or low regularisation in the title of the plot

Reply:
"Stronger regularization (C=0.003)"

and

"Weaker regularization (C=1)"

Reply:
Thanks! Done!